File: testdata/lang/comments.cs
class Comments {
    string text = "// not a comment /* either */";
    string verbatim = @"/* still text */ // still text";
    string raw = """/* still text */ // still text""";
    int value = 1; // line comment
    /* block comment
       second line */
    int next = 2;
}

File: testdata/lang/comments.js
#!/usr/bin/env node
const text = "// not a comment /* either */";
const template = `/* still text */ // ${"still text"}`;
const regex = /\/\* not a comment \*\//;
const value = 1; // line comment
/* block comment
   second line */
<!-- html comment
const next = 2;
